homeassistant/components/wink/climate.py

```diff
@@ -436,7 +440,7 @@ def hvac_mode(self) -> str:
 
         wink_mode = self.wink.current_mode()
         if wink_mode == "auto_eco":
-            return HVAC_MODE_AUTO
+            return HVAC_MODE_COOL
```

Comment:
I think that we should keep returning AUTO here. We could optionally check if AUTO is supported first (it's part of `hvac_modes`)

Reply:
The idea here was to have the "auto_eco" wink mode displayed as "Cool - Eco". If this returns AUTO it will be displayed as "Auto - Eco". Which style is preferred?

I think the AUTO(in hvac_modes) state is causing confusion because the state map is currently shared with the Wink Thermostat which has a real auto mode(e.g. automatic temperate changes). Maybe it's better to separate the state maps for both the Thermostat and Air conditioner for clarity?

Reply:
Oh, this is my bad. I did not realize that this was changing code inside the AC entity. This is ok .